test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -646,7 +646,7 @@ public void TestLdaTransformEmptyDocumentException()
 
             try
             {
-                var lda = new LdaTransform(Env, args, srcView);
+                var lda = LdaTransform.Create(Env, args, srcView);
```

Comment:
>LdaTransform [](start = 26, length = 12)

Can you switch to estimator call?
Basically var lda = LdaEstimator(env, cols).Fit(srcView).Transform(scrView);